### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
- [DEMO LINK](https://maksym-nemera.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://maksym-nemera.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,69 @@
...
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+
```
Comment (naziamoff) [inline src/index.html:17]: no need to add empty line between head and body
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,69 @@
...
+          <h5 class="card__cost">
+            $2,199
+          </h5>
+        </div>
+
+        <a
+        href="#"
+        data-qa="hover"
+        class="card__button"
+        >
```
Comment (naziamoff) [inline src/index.html:63]: don't forget about the indents between elements of different nesting levels
```suggestion
        <a
          href="#"
          data-qa="hover"
          class="card__button"
        >
```

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+.card__review,
+.card__price,
+.card__cost {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
```
Comment (naziamoff) [inline src/styles/blocks/card.css:12]: you can just add it to card and it will be inherited

File: src/styles/blocks/card.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+  --background-color: #fff;
+  --color-code-and-price: #616070;
+  --background-button: #00acdc;
+
+  --line-height-small: 14px;
+  --font-size-small: 10px;
+  --font-weight-small: 400;
+  --font-weight-big: 700;
+}
```
Comment (naziamoff) [inline src/styles/blocks/card.css:32]: 🔥 🔥

**Review by naziamoff — CHANGES_REQUESTED** [review]
already great, but let's make it perfect

File: src/styles/utils/reset.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```
[the remark this answers is not in the document]
Comment (vakopov) [reply src/styles/utils/reset.css:7]: good job

File: src/styles/utils/reset.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
```
Comment (vakopov) [inline src/styles/utils/reset.css:15]: nothing to add

**Review by vakopov — COMMENTED** [review]
👌🏽

**Review by vakopov — APPROVED** [review]

**Pull request closed without merging**